pootle/apps/pootle_app/forms.py

```diff
@@ -75,7 +75,7 @@ def __init__(self, *args, **kwargs):
 
         def clean_localfiletype(self):
             value = self.cleaned_data.get('localfiletype', None)
-            if not value:
+            if value is None:
```

Comment:
This is a valid concern in this context (form field values of character types; their `to_python()` will return `''` for empty values), and probably the best bet is to replace `None` with `''` as the default in the `.get()` call.